recipes/ftxui/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+import os
+
+import conan.tools.files
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class FTXUIConan(ConanFile):
+    name = "ftxui"
+    description = "💻 C++ Functional Terminal User Interface."
```

Comment:
```suggestion
    description = "💻 C++ Functional Terminal User Interface."
```

Is it allowed in conan recipe?